Scheduler/v7/maintel_fbs_sit_block_t345.yaml

```diff
@@ -0,0 +1,47 @@
+maintel:
+  driver_type: feature_scheduler
+  mode: ADVANCE
+  startup_type: COLD
+  startup_database: /home/saluser/rubin_sim_data/fbs_observation_database.sql
```

Comment:
This should be `/home/saluser/rubin_sim_data/fbs_observation_database_maintel.sql`